README.md

```diff
@@ -3,10 +3,17 @@ Pancake
 
 > Pancake is a tool to make working with npm on the front end easy and sweet.
 
+![The Pancake tool](https://raw.githubusercontent.com/govau/pancake/master/assets/pancake.png)
+
+[Npm wrote about](http://blog.npmjs.org/post/101775448305/npm-and-front-end-packaging) the challenges frontend developers face when trying to use npm. Pancake
+is addressing those by embracing the idea of small individually versioned independent modules. Interdependencies is what npm does really well and Pancake
+will help you keep them flat and error out on conflicts. [Read more about our solution](https://medium.com/dailyjs/npm-and-the-front-end-950c79fc22ce)
```

Comment:
Great improvement! After this last line it would be good if it can be more clearly stated exactly what pancake does in an ordered list e.g. 1. imports sass. 2. manages dependencies 3... etc

Reply:
Thanks for the feedback that makes sense